ansible-ipi-install/roles/node-prep/tasks/70_enabled_fw_services.yml

```diff
@@ -1,19 +1,52 @@
 ---
-- name: Enable Firewall for HTTP
-  firewalld:
-    service: http
-    permanent: yes
-    state: enabled
-  become: yes
+- name: Configure firewalld
+  block:
+    - name: Enable Firewall for HTTP
```

Comment:
I know this is nit but the task names look odd here when you run the play because you see the same task names multiple times. Here is an example when you don't have iptables but firewalld

```
TASK [node-prep : Enable Firewall for HTTP] ********************************************************************************
Tuesday 11 February 2020  20:01:20 -0600 (0:00:00.157)       0:03:00.174 ****** 
ok: [worker-0.kni4.cloud.lab.eng.bos.redhat.com]

TASK [node-prep : Open Firewall port 5000/tcp, zone libvirt and public] ****************************************************
Tuesday 11 February 2020  20:01:25 -0600 (0:00:04.766)       0:03:04.940 ****** 
ok: [worker-0.kni4.cloud.lab.eng.bos.redhat.com] => (item=libvirt)
ok: [worker-0.kni4.cloud.lab.eng.bos.redhat.com] => (item=public)

TASK [node-prep : Enable Firewall for HTTP] ********************************************************************************
Tuesday 11 February 2020  20:01:34 -0600 (0:00:08.763)       0:03:13.703 ****** 
skipping: [worker-0.kni4.cloud.lab.eng.bos.redhat.com]

TASK [node-prep : Open Firewall port 5000/tcp] *****************************************************************************
Tuesday 11 February 2020  20:01:34 -0600 (0:00:00.076)       0:03:13.780 ****** 
skipping: [worker-0.kni4.cloud.lab.eng.bos.redhat.com]

TASK [node-prep : Allow related and established connections] ***************************************************************
Tuesday 11 February 2020  20:01:34 -0600 (0:00:00.077)       0:03:13.858 ****** 
skipping: [worker-0.kni4.cloud.lab.eng.bos.redhat.com]

TASK [node-prep : Save iptables configuration] *****************************************************************************
Tuesday 11 February 2020  20:01:34 -0600 (0:00:00.076)       0:03:13.934 ****** 
skipping: [worker-0.kni4.cloud.lab.eng.bos.redhat.com]

TASK [node-prep : include_tasks] *******************************************************************************************
Tuesday 11 February 2020  20:01:34 -0600 (0:00:00.075)       0:03:14.009 ****** 
included: /home/rlopez/git_projects/baremetal-deploy/ansible-ipi-install/roles/node-prep/tasks/80_libvirt_pool.yml for worker-0.kni4.cloud.lab.eng.bos.redhat.com
```

Notice how you see "Enable Firewall for HTTP" and then 2 tasks later you see it again "Enable Firewall for HTP" but its skipped. I'd almost prefer it say for example "Enable {{ firewall }} for HTTP" -- this should translate to the appropriate value since the firewall variable is a fact.

Reply:
Fixed.